### ISSUE TITLE
Error running playbook -> I believe community.general.docker_login moved to community.docker.docker_login

### ISSUE THREAD

**Issue by ieugen** [JoelLefkowitz/swarmroles#2]
Hi,

I get this error:
 
```
ERROR! couldn't resolve module/action 'community.general.docker_login'. This often indicates a misspelling, missing collection, or incorrect module path.

ansible/collections/ansible_collections/joellefkowitz/swarmroles/roles/swarm_manager/tasks/main.yml': line 8, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

- name: Log into DockerHub
  ^ here
```

**Comment by JoelLefkowitz** [comment]
Merged in #3. If this runs properly on your local machine now, I can publish on the patch ansible galaxy.

**Comment by ieugen** [comment]
I've tested this locally and it works for me.
